src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days < 3) {
+    return totalCost;
+  } else if (days < 7) {
+    totalCost = totalCost - 20;
+
+    return totalCost;
+  } else {
+    totalCost = totalCost - 50;
+
+    return totalCost;
```

Comment:
Prefer using `const` over `let` for `totalCost` since you are not reassigning it, but instead returning its modified value directly. This adheres to checklist item 3.